egs/wsj/s5/steps/online/nnet2/extract_ivectors.sh

```diff
@@ -277,4 +277,5 @@ if [ $stage -le 5 ]; then
   for j in $(seq $nj); do cat $dir/ivector_online.$j.scp; done >$dir/ivector_online.scp || exit 1;
 fi
 
-echo "$0: done extracting (pseudo-online) iVectors"
+
+cp $srcdir/ivector_extractor_id $dir 2>/dev/null
```

Comment:
Because this is the last command in the script it determines the exit status.  So you should 'exit 0' after this.
And I don't know why you removed the echo statement above. [note: the echo statement should probably mention $dir].